### PR TITLE
Admin access only for dev/stage cluster

### LINKED ISSUE #63: Read only access to live cluster
Right now, all members of the mindtastic dev team get full access to the kubernetes cluster.
We want to give only read-only access to the cluster to all team members when going through the teleport proxy.
We should also restrict the access more for external members (as a follow-up to #45), so they are not allowed to e.g. read secrets from the cluster.

### PR DESCRIPTION
**Pull request by jgraeger**
Fixes #63

### CODE REVIEW

File: clusters/dev/runtime.tf [hunk 1]
```diff
@@ -1,9 +1,10 @@
 module "teleport" {
   source = "../../modules/teleport-agent"
 
-  cluster_name           = local.cluster_name
-  teleport_proxy_address = var.runtime_teleport_proxy_address
-  teleport_auth_token    = var.runtime_teleport_auth_token
+  cluster_name                 = local.cluster_name
+  teleport_proxy_address       = var.runtime_teleport_proxy_address
+  teleport_auth_token          = var.runtime_teleport_auth_token
+  kubernetes_environment_label = "dev"
```
Comment (ldb) [inline clusters/dev/runtime.tf:7]:
```suggestion
  kubernetes_environment_label = var.environment
```

Comment (ldb) [reply clusters/dev/runtime.tf:7]: Same for the others

File: modules/teleport-cluster/rbac-config.yaml [hunk 2]
```diff
@@ -1,17 +1,17 @@
 kind: role
 version: v5
 metadata:
-  name: k8s-admin
+  name: k8s-dev-admin
```
Comment (ldb) [inline modules/teleport-cluster/rbac-config.yaml:4]:
```suggestion
  name: k8s-developer-admin
```

not to confuse with `dev` environment.

**Review by ldb — COMMENTED** [review]
Just some small things

**Review by akkbng — APPROVED** [review]
Looks great

**Review by ldb — APPROVED** [review]

**Pull request merged**